contrib/init/sysvinit-debian/docker

```diff
@@ -30,6 +30,7 @@ DOCKER_SSD_PIDFILE=/var/run/$BASE-ssd.pid
 DOCKER_LOGFILE=/var/log/$BASE.log
 DOCKER_OPTS=
 DOCKER_DESC="Docker"
+UNSHARE_BINARY=${UNSHARE_BINARY:-/usr/bin/unshare}
```

Comment:
This one should probably just be `UNSHARE` to match `DOCKER` (where the openrc one actually has `DOCKER_BINARY` so `UNSHARE_BINARY` there is consistent).